src/main/g8/scalafix/project/plugins.sbt

```diff
@@ -1,4 +1,3 @@
 addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "$scalafix_version$")
-addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-RC9")
-addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.6.1")
-
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.2")
```

Comment:
Pro tip, `coursier.util.Properties.version` can be used to avoid copy-pasting 1.0.2 😉